ios/RNSScreen.m

```diff
@@ -196,6 +197,12 @@ - (void)setStatusBarHidden:(BOOL)statusBarHidden
   }
 }
 
+- (void)setStackOrientationMask:(RNSScreenOrientationMask)stackOrientationMask
+{
+  _stackOrientationMask = stackOrientationMask;
```

Comment:
Commented about it below. What is the point of having our own type for orientation if the only thing we use it for is to convert to UIKit compatible type. This seem to add a lot of boilerplate